app/domain/processor.js

```diff
@@ -37,23 +55,29 @@ const processGrants = async (grants, manifestGrants, schemeName, containerClient
       })
 
       for (const [index, chunk] of chunks.entries()) {
-        const chunkTitle = `${hash}_${index + 1}.txt`
+        console.log(`Processing chunk ${index + 1}/${chunks.length}...`)
```

Comment:
console.info maybe?
To separate the helpful dev consoles from the console.logs that we might be cleaning up in the future